analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -58,6 +58,7 @@ import org.ossreviewtoolkit.model.createAndLogIssue
 import org.ossreviewtoolkit.model.yamlMapper
 import org.ossreviewtoolkit.utils.common.Os
 import org.ossreviewtoolkit.utils.common.collectMessages
+import org.ossreviewtoolkit.utils.common.getCommonParentFile
```

Comment:
Commit message nit: "simply" is probably a bit overstated given that the code became a lot more complex.